src/components/dashboard/TrialBanner.vue

```
@@ -16,6 +16,8 @@ import type { ComponentPublicInstance } from 'vue'
 import { computed, onUnmounted, ref, watch } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { useOrganizationStore } from '~/stores/organization'
+import { pushEvent } from '~/services/posthog'
```

ESLint

[error] Expected "~/services/posthog" to come before "~/stores/organization".

(perfectionist/sort-imports)